pants.toml

```diff
@@ -2,10 +2,11 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 [GLOBAL]
-pants_version = "1.28.0"
+pants_version = "1.29.0rc0"
 v1 =  false  # Turn off the v1 execution engine.
 v2 = true  # Enable the v2 execution engine.
-v2_ui = true  # Enable the v2 execution engine's terminal-based UI.
+dynamic_ui = true
```

Comment:
This defaults to enabled/auto now, yea? Shouldn't need to pass it.

Reply:
Indeed. Setting V2 = true is also unnecessary. Benjy thought we should still do it to make it transparent to the user that the option exists. 

But I think I agree both V2 and dynamic_ui should just use the defaults? Fwit, we show how to explicitly turn off the dynamic UI in the CI page of the docs already.